src/script/lua_api/l_rollback.cpp

```diff
@@ -80,6 +83,14 @@ int ModApiRollback::l_rollback_revert_actions_by(lua_State *L)
 	int seconds = luaL_checknumber(L, 2);
 	Server *server = getServer(L);
 	IRollbackManager *rollback = server->getRollbackManager();
+
+	// If rollback is disabled, tell it's not a success.
+	if (rollback == NULL) {
+		bool success = false;
+		lua_pushboolean(L, success);
```

Comment:
The success variable is pointless, just do `lua_pushboolean(L, false);  // Success` or so.